### DISCUSSION TITLE
V2.0 question

### DISCUSSION THREAD

**Discussion by DigiEggz** [jidoc01/RustyDO#14] [General]
Has anyone seen any mention about first-hand experience with the V2.0 beta? Was it really publicly released? I've seen various articles detailing a new card collection mode. When I was parsing through the client, I saw some story-like dialogue and it made me wonder if it was connected to this new mode, perhaps as unfinished content. I even saw mention of 8 new characters being added, but I'm not sure if this is accurate.

Any extra information or details about the beta would be helpful!

**Comment by jidoc01** [comment]
There is a news article that mentions the closed-beta test (CBT) of V2.0: https://www.gamemeca.com/view.php?gid=49211

It seems that the V2.0 client was distributed to a small number of testers, and I think that is why we hardly find any information about V2.0 😢.

FYI, the currently archived client (V1.5) was also really hard to obtain since it came from some non-Korean Digimon fan who was abroad. And I had to search it on Google to find it out.

You may find some stories about DO in its Korean wiki page: https://namu.wiki/w/%EB%94%94%EC%A7%80%EB%AA%AC%20%EC%98%A8%EB%9D%BC%EC%9D%B8

**Reply by DigiEggz** [reply]
I found some images of the new card collection feature and the exploration mode that was planned for V2.0, so I know those were likely in the beta version.

![bkg](https://github.com/jidoc01/RustyDO/assets/3136062/aee4182a-311f-445b-b84e-37dfe1292f00)
 
There were several mentions of extra characters being added as well, but I have yet to see any screenshots of new characters. It makes me wonder if it was merely planned or if the characters were available in 2.0, but no images exist. I'll continue to look around.

Is it possible to ask around on Korean message boards to see if anyone played the beta or had a friend that did? It's entirely possible someone, somewhere still has files backed up on an old drive or disc. I found an [earlier version](https://drive.google.com/file/d/1LVl8MurEBPmiUBWPXB0-Xsr90kW5U2u1/view?usp=sharing) that's a bit older the one we have. 

999 players were selected for the V2.0 beta. That's not a lot of people, but at the same time, one thousand people isn't a small number either. I'll try to remain hopeful!
